app/lib/server/startup/settings.js

```diff
@@ -998,6 +998,34 @@ settings.addGroup('Message', function() {
 			public: true,
 			i18nDescription: 'Message_Attachments_GroupAttachDescription',
 		});
+
+		this.add('Message_Attachments_Thumbnails_Enabled', true, {
+			type: 'boolean',
+			public: true,
+			i18nDescription: 'Message_Attachments_Thumbnails_EnabledDesc',
+		});
+
+		this.add('Message_Attachments_Thumbnails_Width', 200, {
+			type: 'int',
+			public: true,
+			enableQuery: [
+				{
+					_id: 'Message_Attachments_Thumbnails_Enabled',
+					value: true,
+				},
+			],
+		});
+
+		this.add('Message_Attachments_Thumbnails_Height', 200, {
```

Comment:
same for height (https://github.com/RocketChat/Rocket.Chat/blob/3b003da3dc403764b293aa02709af23c7256623b/client/components/Message/Attachments/context/AttachmentContext.tsx#L18):
```suggestion
		this.add('Message_Attachments_Thumbnails_Height', 360, {
```